keyboards/annepro2/keymaps/iso_default/config.h

```diff
@@ -0,0 +1,20 @@
+ /* Copyright 2022 OpenAnnePro community
+  *
+  * This program is free software: you can redistribute it and/or modify
+  * it under the terms of the GNU General Public License as published by
+  * the Free Software Foundation, either version 2 of the License, or
+  * (at your option) any later version.
+  *
+  * This program is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+#pragma once
+
+#undef PRODUCT
+// C18 is a macro
+#define PRODUCT Anne Pro 2 c18-iso (QMK)
```

Comment:
Just a note, this will need to be wrapped in double quotes in develop (https://github.com/qmk/qmk_firmware/pull/18183), or it will fail to compile. There is no change needed here.